pkg/placement/workload-projector.go

```diff
@@ -257,16 +205,12 @@ func NewWorkloadProjector(
 			obj = dfu.Obj
 		}
 		syncfg := obj.(*edgeapi.SyncerConfig)
-		cluster := logicalcluster.From(syncfg)
 		if syncfg.Name != SyncerConfigName {
```

Comment:
This is wrong, should have been fixed in #1265 (@waltforme ). The projected object in the KCS does not have a constant name, the name has been mangled to include kube-bind's identifier for the MBS.